multicluster/controllers/multicluster/stale_controller_test.go

```diff
@@ -418,3 +420,147 @@ func TestStaleController_CleanupClusterInfoImport(t *testing.T) {
 		})
 	}
 }
+
+func TestStaleController_CleanupMemberClusterAnnounce(t *testing.T) {
+	tests := []struct {
+		name                              string
+		memberClusterAnnounceList         *mcsv1alpha1.MemberClusterAnnounceList
+		clusterSet                        *mcsv1alpha1.ClusterSetList
+		exceptMemberClusterAnnounceNumber int
+	}{
+		{
+			name:                              "no MemberClusterAnnounce to clean up",
```

Comment:
You shouldn't give two test cases with the same name, it will be hard to debug when there is a failure.
```suggestion
			name:                              "no MemberClusterAnnounce to clean up when there is no resource",
```